src/components/pageTemplate/InputTemplate.tsx

```diff
@@ -0,0 +1,12 @@
+import { Input } from "antd";
+import { SearchOutlined } from "@ant-design/icons";
+
+interface inputTemplateProps {
+    onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const InputTemplate: React.FC<inputTemplateProps> = ({ onChange }) => {
+    return <Input placeholder='Search' onChange={onChange} prefix={<SearchOutlined />} />;
+};
```

Comment:
please use global FormTextField component instead

Reply:
ofc, but your merge it after my PR